### PR TITLE
Instructions on installing Pop!_gtk theme

### PR DESCRIPTION
**Pull request by ryanleesipes**
Included are the initial instructions for install the Pop!_gtk theme. We should add more information on how to install on other distributions, if and when packages for those are created and well maintained.

### CODE REVIEW

**Comment by WatchMkr** [comment]
Needs proper title and description like...
---
title: Convert Ubuntu to Pop!_OS
description: Learn how to turn Ubuntu 17.04+ into Pop!_OS
---

Also list all the packages the meta package pulls in at..... "However, individual componenets can be installed separately, e.g.:

sudo apt install pop-gtk-theme
"

"components" typo in the line above.

Finally, use the convention Pop!_Theme (capital project name).

**Comment by ryanleesipes** [comment]
@cassidyjames I've incorporated @WatchMkr's feedback. Please review.

File: docs/install-pop-theme.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: Install the Pop!_theme
```
Comment (cassidyjames) [inline docs/install-pop-theme.md]: `Pop!_Theme`

Comment (cassidyjames) [inline docs/install-pop-theme.md]: Or "Pop! theme" if it's not really a product name.

**Pull request merged**